ckanext/gitdatahub/plugin.py

```diff
@@ -77,18 +78,31 @@ def after_update(self, context, pkg_dict):
                 json.dumps(body, indent=2),
                 contents.sha
                 )
-            
+            try:
+                lfs_pointers = [obj.name for obj in repo.get_contents("data")]
+            except UnknownObjectException as e:
+                lfs_pointers = list()
+
             gitattributes_body = ''
-            for obj in body['resources']:
+            for index, obj in enumerate(body['resources']):
                 gitattributes_body += "data/{} filter=lfs diff=lfs merge=lfs -text\n".format(obj['title'])
-            
+                if obj['title'] not in lfs_pointers:
+                    sha256 = pkg_dict['resources'][index]['id']
```

Comment:
We don't have these specifications on how are we going to receive the `sha256` yet, but I will assume a `sha256` key instead of the `index`. Since we need to wait for other implementation I will create an issue to remember this. No need to change anything now.